Instastories.py

```diff
@@ -252,7 +252,7 @@ def start_scrape(folder_path, number_of_persons, mode_flag = "all"):
     timestampStr = datetime.datetime.now().strftime("%d-%b-%Y (%H:%M:%S)")
 
     with open("run_history.log", "a+") as o:
-        scraped_users = len(ids)
+        scraped_users = len(ids) if number_of_persons < 0 or number_of_persons > len(ids) else number_of_persons
```

Comment:
min(max(0, number_of_persons), len(ids)) should be equivalent

Reply:
Add a comment explaining why you are doing this.
eg

# number_of_persons is the limit we set, but at most we can scrape len(ids) users.